letsdrill.rb

```diff
@@ -1,12 +1,19 @@
 def get_letter_grade(integer)
  
   #Put your code here!
-
+  case integer
+    when 90..100 then "A"
```

Comment:
Thanks for the advice about 2 spaces guys.  I remember reading that a while back, but habits die hard...thanks!